notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>Adding a quick one line description as the following might be helpful:</p><blockquote>A function to query EFD data of TMA aximuth with a given time period. The queried fields are actual and demand postions and velocities with time stamp.</blockquote><p>or to make it shorter like:</p><blockquote>Same as <code>call_az_data_range</code> but for elevation. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>In elevation</p>